modules/tensor_mechanics/src/materials/IsotropicPlasticityStressUpdate.C

```diff
@@ -53,11 +93,11 @@ IsotropicPlasticityStressUpdate::IsotropicPlasticityStressUpdate(const InputPara
     _yield_condition(-1.0), // set to a non-physical value to catch uninitalized yield condition
     _hardening_slope(0.0),
     _plastic_strain(
-        declareProperty<RankTwoTensor>(_base_name + _plastic_prepend + "plastic_strain")),
-    _plastic_strain_old(
-        getMaterialPropertyOld<RankTwoTensor>(_base_name + _plastic_prepend + "plastic_strain")),
-    _hardening_variable(declareProperty<Real>(_base_name + "hardening_variable")),
-    _hardening_variable_old(getMaterialPropertyOld<Real>(_base_name + "hardening_variable")),
+        declareProperty<RankTwoTensor>(this->_base_name + _plastic_prepend + "plastic_strain")),
```

Comment:
add `using ...::_base_name;` in the header

Reply:
does ```...::_base_name``` work?  I didn't see it in the code.  I used the templated version.